example/create-api-keys.js

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env node
+"use strict";
+
+var container, exampleLib, hash, neodoc;
+
+/**
+ * Returns the usage instructions
+ *
+ * @return {string}
+ */
+function usageHelp() {
+    return `
+Example script to create a set of API keys for OpenToken.
+
+Usage:
+
+    create-api-keys [OPTIONS]
+
+Options:
+
+    --account=ID        Specifies the OpenToken account ID.  This overrides
+                        the use of the OPENTOKEN_ACCOUNT environment variable.
+    --api=URL           Specify the API URL.  This would override the use of
+                        OPENTOKEN_API environment variable.
+    --description=DESC  Set the email address for the account.
+    --help, -h          Show this help message.
+    --mfa=KEY           Assigns the Base32 encoded MFA key.  Using this as
+                        a command-line argument is insecure.  You should prefer
+                        the OPENTOKEN_MFA environment variable when possible.
+    --password=PW       Use the given password.  Putting this on the command
+                        line is not very secure.  You can also use
+                        OPENTOKEN_PASSWORD environment variable.
+`;
+}
+
+
+/**
+ * The main program.
+ */
+function main() {
+    var args, config, description;
+
+    args = neodoc.run(usageHelp());
+    config = exampleLib.config(args);
+
+    if (args["--help"]) {
+        console.log(usageHelp().trim());
+
+        return;
+    }
+
+    if (args["--description"]) {
+        description = args["--description"];
+    } else {
+        description = "";
+    }
+
+    if (!config.password || !config.api || !config.mfa || !config.account) {
+        console.log("You are missing a required setting.  This needs");
+        console.log(" * API URL");
+        console.log(" * Account ID");
+        console.log(" * Account password");
+        console.log(" * MFA key (the TOTP seed, base32 encoded)");
```

Comment:
I put them in this order because that's how they are used.  I put the options alphabetically so they are easy to reference.  I can't say that this list makes more sense.

* Account ID
* API URL
* MFA key (the TOTP seed, base32 encoded)
* Account password

I would imagine I always want the account password next to the the account ID.  Thoughts?